java/code/src/com/redhat/rhn/domain/credentials/CredentialsFactory.java

```diff
@@ -140,6 +140,17 @@ public static Credentials createRegistryCredentials() {
         return creds;
     }
 
+    /**
+     * Helper method for creating new Cloud Rmt {@link Credentials}
+     * @return new credential with type Cloud Rmt
+     */
+    public static Credentials createCloudRmtCredentials() {
```

Comment:
I wonder whether "Cloud RMT" is a good name for this concept. Given you are using "PAYG SSH" elsewhere, will it help to settle on one piece of jargon only?

Reply:
I know it's a new jargon, but it is really the authentication to the cloud rmt infrastructure. 
My idea was to keep the concepts separated, and if in the future we had a new way to obtain those credentials, it will not be tied to the PAYG concept. 
For example, imagine we deploy a SUMA PAYG instance, in which we have access to cloud-rmt network. In that case we can use the same tables and authentication logic and is not tied to the instance PAYG logic.